x509/build/build.rs

```diff
@@ -87,3 +88,33 @@ fn gen_fmc_alias_cert(out_dir: &str) {
     let template = bldr.tbs_template("Caliptra FMC Alias", "Caliptra LDevID");
     CodeGen::gen_code("FmcAliasCertTbs", template, out_dir);
 }
+
+fn gen_rt_alias_cert(out_dir: &str) {
+    let mut usage = KeyUsage::default();
+    usage.set_key_cert_sign(true);
+    let bldr = cert::CertTemplateBuilder::<EcdsaSha384Algo>::new()
+        .add_basic_constraints_ext(true, 0)
```

Comment:
I think we can make the pathLenConstraint 1 (Runtime is only expected to sign DPE certs which are leaf certs).